RUFAS/biophysical/feed_storage/feed_manager.py

```diff
@@ -311,7 +311,11 @@ def get_total_inventory(self, inventory_date: date, weather: Weather, time: Rufa
 
         available_feed_rufas_ids = [feed.rufas_id for feed in self._available_feeds]
 
-        available_feed_totals = self._query_available_feed_totals(available_feed_rufas_ids, projected_crops)
+        projected_shrinkage = self.purchased_feed_storage.project_shrinkage(days_in_the_future, self._available_feeds)
+
+        available_feed_totals = self._query_available_feed_totals(
```

Comment:
I think this function would do the work to prevent any purchased feeds that are not part of the available feeds from making it to the diet (referencing an earlier comment about not needing to send the available feeds to check against when projecting shrink for purchased feeds in storage).